src/TypeSpec.Extension/Emitter.Csharp/src/type/inputType.ts

```diff
@@ -62,7 +63,7 @@ export interface InputEnumType extends InputType {
     Namespace?: string;
     Accessibility?: string;
     Deprecated?: string;
-    Description: string;
+    Description?: string;
```

Comment:
Is this change related to the support of addtionalProperties?

Reply:
No this is a clean up to align with the definitions.